xbrowse_server/base/management/commands/add_individuals_to_project.py

```diff
@@ -14,25 +14,24 @@ class Command(BaseCommand):
         make_option('--sample-list'),
         make_option('--vcf'),
         make_option('--ped'),
-        make_option('--create_phenotips_patients'),
-        make_option('--phenotips_username'),
-        make_option('--phenotips_password'),
+        make_option('--create_phenotips_patients',
+                    action='store_true',
+                    dest='create_phenotips_patients',
+                    default=False,
+                    help='add this patient to phenotips'),
     )
 
     def handle(self, *args, **options):
       
-        if options['vcf'] is None or options['ped'] is None or len(args)==0:
-          print '\n\nPlease enter a VCF file (--vcf), a PED file (--ped), and a project ID (first positional argument)'
-          print 'for example: python manage.py add_individuals_to_project  myProjectId  --vcf myVcf.vcf  --ped myPed.ped\n'
-          print 'If you also wish to create patients in phenotips, please use the --create_phenotips_patients,--phenotips_username, --phenotips_password to do so. ALL of them are REQUIRED.\n'
+        if len(args)==0 or options['vcf'] is None and options['ped'] is None:
+          print '\n\nPlease enter a VCF file (--vcf), OR IDEALLY A PED file (--ped), and a project ID (first positional argument)'
```

Comment:
Very minor point, but for consistency we should chose if error messages and method docs start with upper case or lower case. (eg. Please vs. please). I would vote for upper case.